### DISCUSSION TITLE
11-29

### DISCUSSION THREAD

**Discussion by hanseulhee** [ChristmasTest/christmas#21] [Daily_Meeting]
### 🎅🏻 회의 내용

스프린트 두번째 개발 내용 공유

윤혁님 - bgm (소리를 켰을 때 벨이 움직이는 등 음악이 흘러나온다는 게 보여지면 좋을 듯)
슬희님 - sentry
수영님 - seo (metadata.tsx -> metadata.ts)
수님 - 얼렁뚱땅뚱땅 (많이함)

### ❄️ 오늘 할일

- [x] 역할 분담

12/2 까지 스토리북 master 고'수'가 되기

**Comment by oauch** [comment]
뚱땅뚱땅 수님 〰️
